### PR TITLE
 DIV-6525-Updating-document-names-to-follow-convention

### PR DESCRIPTION
**Pull request by madjava**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DIV-6525

### Change description ###

Refusal document names start with uppercase others start lowercase. Updating to follow convention

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by yarpo — APPROVED** [review]
Please make sure COS will still work

File: definitions/divorce/json/FixedLists/FixedLists.json [hunk 1]
```diff
@@ -1266,18 +1266,6 @@
     "ListElement": "Cheque",
     "DisplayOrder": 4
   },
-  {
-    "LiveFrom": "23/07/2020",
-    "ID": "d8documentsReceivedEnum",
```
Comment (Dizrupto) [inline definitions/divorce/json/FixedLists/FixedLists.json:1271]: Wait, was that released already?

Comment (madjava) [reply definitions/divorce/json/FixedLists/FixedLists.json:1271]: @Dizrupto Good question. When was our last release, maybe safer to leave where it was. Doesnt hurt just though to make the feature more obvious.

**Review by Dizrupto — APPROVED** [review]

**Comment by madjava** [comment]
Happy to merge this PR

**Pull request merged**